.github/workflows/linting.yml

```diff
@@ -29,4 +29,16 @@ jobs:
           submodules: 'recursive'
       - name: Lint with flake8
         run: |
-          make lint
+          make lint_python
+
+  lint_ruby:
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 12
```

Comment:
curious why we limit this?

Reply:
We don't have a specific reason but we just show 12 as a reasonable number. If you think we need to change that. Please let me know and propose a number.

Reply:
Is the default not good enough?

Reply:
It's good.

Should I remove it ?

Reply:
Yes, if the default is good enough no reason to override it :)

Reply:
Done